### PR TITLE
Fix appveyor ci

### PR DESCRIPTION
**Pull request by tomap**
wrong step name

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -34,12 +34,12 @@ test_script:
   - npm run eslint
   - npm run test-cov
 
-after_script:
+after_test:
   - npm install coveralls
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
-
+  - node ./node_modules/coveralls/bin/coveralls.js < ./coverage/lcov.info
```
Comment (curbengh) [inline appveyor.yml]: I think appveyor shouldn't run coveralls, otherwise duplicate results can be confusing.

Comment (tomap) [reply appveyor.yml]: I dropped it (commented, just in case)

File: appveyor.yml [hunk 2]
```diff
@@ -34,12 +34,12 @@ test_script:
   - npm run eslint
   - npm run test-cov
 
-after_script:
-  - npm install coveralls
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
-
+# after_test:
```
Comment (curbengh) [inline appveyor.yml]: I think we should remove the lines altogether, hexo doesn't have them either.

Comment (tomap) [reply appveyor.yml]: Done

**Pull request merged**